### PR TITLE
Promote filestore v1.4.1 to k8s-prod

### PR DESCRIPTION
**Pull request by tyuchn**
[v1.4.1 image](https://pantheon.corp.google.com/gcr/images/k8s-staging-cloud-provider-gcp/global/gcp-filestore-csi-driver@sha256:2249116ead26d7a857deb659aef4eceef927374d18df89a613525b3eb167f49e/details?jsmode=o&mods=logs_tg_prod&project=k8s-staging-cloud-provider-gcp&supportedpurview=project)

### CODE REVIEW

**Review by leiyiz — COMMENTED** [review]
/lgtm
/approve

File: k8s.gcr.io/images/k8s-staging-cloud-provider-gcp/images.yaml
Comment (BenTheElder) [inline k8s.gcr.io/images/k8s-staging-cloud-provider-gcp/images.yaml:1]: you need to do this same PR but under registry.k8s.io/, all PRs to k8s.gcr.io/ are blocked and these manifests will eventually be phased out entirely.

see: https://github.com/kubernetes/enhancements/issues/3720

Comment (BenTheElder) [reply k8s.gcr.io/images/k8s-staging-cloud-provider-gcp/images.yaml:1]: https://github.com/kubernetes/k8s.io/pull/4967#issuecomment-1472941717

**Comment by msau42** [comment]
reopened in https://github.com/kubernetes/k8s.io/pull/4978

/close

**Pull request closed without merging**